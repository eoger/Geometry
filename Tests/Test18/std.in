R Rect 4 4 4 4
C Cercle 5 4 3
L Ligne 1 2 3 4
PL PolyL 1 2 3 4 5 6
OA oa1 Cercle Ligne PolyL
OA oa2 oa1 PolyL Rect
OA oa3 oa1 oa2
LIST
CLEAR
LIST
EXIT