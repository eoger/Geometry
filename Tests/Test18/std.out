OK
OK
OK
OK
OK
OK
OK
C Cercle 5 4 3
L Ligne 1 2 3 4
OA oa1 Cercle Ligne PolyL
OA oa2 PolyL Rect oa1
OA oa3 oa1 oa2
PL PolyL 1 2 3 4 5 6
R Rect 4 4 4 4
OK


